engine/src/main/scala/cromwell/engine/workflow/lifecycle/WorkflowExecutionActor.scala

```diff
@@ -62,57 +120,58 @@ final case class WorkflowExecutionActor(workflowId: WorkflowId, workflowDescript
     *      failed to start
     */
   private def startJob(jobKey: BackendJobDescriptorKey,
-                       inputs: Map[FullyQualifiedName, WdlValue],
+                       symbolCache: SymbolCache,
                        configDescriptor: BackendConfigurationDescriptor,
-                       factory: BackendLifecycleActorFactory): WorkflowExecutionActorState = {
-    val jobDescriptor = BackendJobDescriptor(workflowDescriptor.backendDescriptor, jobKey, inputs)
-    val jobExecutionActor = context.actorOf(
-      factory.jobExecutionActorProps(
-        jobDescriptor,
-        BackendConfigurationDescriptor(configDescriptor.backendConfig, configDescriptor.globalConfig)
-      ),
-      s"$workflowId-BackendExecutionActor-${jobDescriptor.key.tag}"
-    )
-    jobExecutionActor ! ExecuteJobCommand
-    WorkflowExecutionInProgressState
-  }
-
-  private def startJob(jobKey: BackendJobDescriptorKey, inputs: Map[FullyQualifiedName, WdlValue]): WorkflowExecutionActorState = {
+                       factory: BackendLifecycleActorFactory): Try[ExecutionStatus] = {
+
+      val call = jobKey.call
+      fetchLocallyQualifiedInputs(jobKey, symbolCache) match {
```

Comment:
@Horneth Question: Do we need to use this function `fetchLocallyQualifiedInputs` or will it work if we just convert the entries in the SymbolCache and pass it as a Map[FQZ -> WdlValue] to the backend?